### ISSUE TITLE
[Bug]: (GraphQL) key within "_editableName" property doesn't match order defined in headless template (within block editable)

### ISSUE THREAD

**Issue by basecom-de** [pimcore/data-hub#493]
### Expected behavior

Using the following headless template:

```
layout: simple
group: null
content:
    testBlock:
        type: block
        label: testBlock
        config:
            limit: 10
            reload: false
            default: 0
        items:
            testInput:
                type: input
                label: testInput
```
create the following headless template content:

<img width="848" alt="Bildschirmfoto 2022-03-11 um 09 24 58" src="https://user-images.githubusercontent.com/29371208/157830156-216f4a76-bb15-45b2-8aea-ea6088977062.png">

reading the document editables via a DataHub connection (of the Type GraphQL) will result in the following output:

<img width="1561" alt="Bildschirmfoto 2022-03-11 um 09 07 22" src="https://user-images.githubusercontent.com/29371208/157830283-624fe510-6f6c-48e6-8830-e2e8b0d63038.png">

**Expected behaviour would be that the key within the "_editableName"  property would match their defined order**

### Actual behavior

See above

### Steps to reproduce

See above

**Comment by mcop1** [comment]
@basecom-de Thanks, I was able to verify your problem. But it seems to be a general problem and nothing related to GraphQL/Datahub.
I created a new issue in the pimcore repo https://github.com/pimcore/pimcore/issues/11685 and we will keep this one open until it gets fixed there.

**Comment by mcop1** [comment]
@basecom-de we took a closer look at this issue in today´s planning session.
GraphQL returns the editables in the correct order - like they are ordered in the headless document itself. 

I guess you are using the first green add ("+") button to add the three inputs and after that you assign the label to the corresponding input. If you add an input and assign the label right away it gets clearer what actually happens:

1.) First input
![image](https://user-images.githubusercontent.com/89011527/159284196-1b8f99aa-c9de-4ef1-8120-80bea28cf802.png)
2.) Second input
![image](https://user-images.githubusercontent.com/89011527/159284267-531589dc-2bf5-4fd9-99f0-e66d861b8690.png)
3.) so far so good .. but third input ...
![image](https://user-images.githubusercontent.com/89011527/159284431-950acdb2-b869-44c0-b87b-62b25907eb8b.png)

So basically the add button always adds a new editable right after the editable you click the button on. Not at the end of the page. The name does not necessarily reflect the current order of the document. 

Best regards

**Comment by basecom-de** [comment]
Thank you. I wasn't aware that the integer encoded within the '_editableName' property references the key instead of the index of the editable.

The order of the keys can be read by accessing the associated block editable and reading out the indices property (which is an array that lists the keys in the order defined by the specific headless document)
